.template/addons/docker/docker-compose.test.yml.tt

```diff
@@ -2,27 +2,29 @@ version: '3.2'
 
 services:
   db:
-    image: postgres:12.1
+    image: postgres:<%= POSTGRES_VERSION %>
     container_name: <%= APP_NAME %>_db
     environment:
       - POSTGRES_DB=<%= APP_NAME %>_test
     ports:
       - "5432"
 
   redis:
-    image: redis:5.0.7
+    image: redis:<%= REDIS_VERSION %>
     container_name: <%= APP_NAME %>_redis
     ports:
       - "6379"
 
   test:
     build:
-      context: .
+      context: ../../..
```

Comment:
This looks strange as it should build from the root directory 🤔

Reply:
I think the rubymine automatically updated it after I move the file around. It should be `.` like the old one 👍